### ISSUE TITLE
The flamer is too weak for its price after recent nerfs

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/gameplay#51]
Since the last nerfs of flamer, the weapon became completely unusable in practice.

**Comment by bmorel** [comment]
This issue was moved from unvanquished's tracker, so this is why it does not follows the template.
Let me rephrase it better:

In 0.53, flamer received several nerfs:

* it's damages were reduced a lot
* it's chances to put things on fire were reduced a lot

The consequences of those are that, *in my opinion*, the flamer became so useless we could just remove it from the game and it would hardly make a difference. It would actually make the humans stronger, since bots would no longer be able to buy it.

**Comment by Viech** [comment]
I warned repeatedly that whatever you do to nerf _fires_, you should not remove the high damage from the flamer as that would make it useless. :stuck_out_tongue: 

So I can pretty much agree here without having played the game recently. The flamer was previously borderline bad against all aliens but dretches and tyrants and was only usable as it was just good enough to keep aliens at a distance for long enough to maybe enter the base and wreck some havoc.

**Comment by DolceTriade** [comment]
Bumping this.

I agree the flamer is quite useless.

My ideas to make the flamer more useful are:
 - Give it ammo again
 - Bump damage
 - Allow it to burn aliens continuously (Damage over time) via burn damage if burned for X seconds
 - Burn damage will stop if aliens get to a booster

I think flamer should basically be a dretch repellent.

**Comment by bmorel** [comment]
> Give it ammo again

Ammo were not reduced, or I do not understand what you mean.
